pkg/helpers/helpers.go

```diff
@@ -51,6 +51,7 @@ func EnsureCRDs(ctx context.Context, client apiextensionsclient.Interface, fs em
 					required,
 				)
 				if err != nil {
+					klog.Errorf("fail to apply %s due to %v", crdFileName, err)
```

Comment:
debug info?

Reply:
yes. it is wait.PollImmediate method, print the error if it has. in case, we cannot know what is exact error.